StackScript/torgap-demo.sh

```diff
@@ -0,0 +1,364 @@
+#!/bin/bash
+
+#  torgap-demo.sh - Installs torgap-demo behind a tor address derived from a minisign secret key.
+#                   Onion service then offers to download and verify a signed text file.
+#
+#  Created by @gorazdko 2020-11-19
+#  
+#  url: https://github.com/BlockchainCommons/torgap-demo
+#  
+#  Based on LinodeStandUp.sh by Peter Denton
+#  source: https://github.com/BlockchainCommons/Bitcoin-Standup-Scripts/blob/master/Scripts/LinodeStandUp.sh
+#
+#
+
+
+
+# It is highly recommended to add a Tor V3 pubkey for cookie authentication.
+# It is also recommended to delete the /standup.log, and
+# /standup.err files.
+
+# torgap-demo.sh sets Tor and torgap-demo as systemd services so that they
+# start automatically after crashes or reboots. If you supply a SSH_KEY in
+# the arguments it allows you to easily access your node via SSH using your rsa
+# pubkey, if you add SYS_SSH_IP's it will only accept SSH connections from those
+# IP's.
+
+# torgap-demo.sh will create a user called standup, and assign the optional
+# password you give it in the arguments.
+
+# torgap-demo.sh will create two logs in your root directory, to read them run:
+# $ cat standup.err
+# $ cat standup.log
+
+# This block defines the variables the user of the script needs to input
+# when deploying using this script.
+#
+# <UDF name="hostname" label="Short Hostname" example="Example: bitcoincore-testnet-pruned"/>
```

Comment:
I'm not sure I understand why hostname and fqdn must be user created here.

Reply:
tnx, these are remnants from the previous script. I removed them now.